site.yml

```diff
@@ -38,5 +38,5 @@
 
 - hosts: veyepar-encoder
   roles:
-  - nfs-client
+      # - nfs-client
```

Comment:
Why are we commenting it out? Either it should be there, or not, or behind a config flag. Comments are about the shittiest option.